loader/proto.go

```diff
@@ -429,10 +550,14 @@ func (b *builder) handleService(s *proto.Service) error {
 						}
 					default:
 						method = n
-						url, err = b.handleLiteralString(*l.Literal)
+						tmp, err := b.handleLiteralString(*l.Literal)
 						if err != nil {
 							return b.formatError(opt.Position, "option for %q should be a string (url)", method)
 						}
+						var re = regexp.MustCompile(`\{(.*?)\}`)
```

Comment:
compile this regex as a global, so you don't have to keep recomipiling it. also, prefer `x := value` over `var x = value` for local variables.